src/reflect/scala/reflect/internal/Names.scala

```diff
@@ -220,7 +220,7 @@ trait Names extends api.Names {
 
     /** Copy bytes of this name to buffer cs, starting at position `offset`. */
     final def copyChars(cs: Array[Char], offset: Int) =
-      scala.compat.Platform.arraycopy(chrs, index, cs, offset, len)
+      java.lang.System.arraycopy(chrs, index, cs, offset, len)
```

Comment:
same, no allocation here.
Unless you decide to remove `Platform.arraycopy` completely, I don't think the changes where there's no allocation, just a copy are worth the trouble :)